package.json

```diff
@@ -107,5 +107,16 @@
     "stylelint-order": "^6.0.1",
     "vinyl": "^3.0.0",
     "wait-on": "^7.0.1"
+  },
+  "overrides": {
+    "postcss-unmq": {
+      "postcss": "^7"
```

Comment:
I've overridden the `postcss` dependency (from v5) to v7

This is the maximum version we can take before the plugin API includes breaking changes, but CSS output remains identical